skyportal/utils/offset.py

```diff
@@ -576,7 +602,7 @@ def get_nearby_offset_stars(
         else:
             offsets = ""
 
-        name = f"{basename}_off{i+1}"
+        name = f"{abrev_basename}_o{i+1}"
```

Comment:
Use the abbreviated name as for the spectroscopic offset stars. These stars are typically not observed, just used to find the source it itself using telescope offsets. So the name does not need to be precise here.